<launch>

    <arg name="camera_name" default="192.168.1.187" />

    <node name="markermapper" pkg="aruco_pkg" type="marker_mapper_node" output="screen" >
        <param name="camerafile" value="$(find camnode)/camera_info/$(arg camera_name).yaml" />
        <param name="outBaseName" value="marker_map" />
        <param name="savePath" value="$(find aruco_pkg)/markermap/" />
        <param name="dictionary" value="ARUCO" />
        <param name="marker_size" value="0.135" />
        <param name="reference_marker_id" value="0" />
        <param name="saveFrameToPCD" value="false" />
        <remap from="/markermapper/image_raw" to="/camnode/image_raw" />
    </node>

    <!--<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen" >
        <remap from="image" to="/map_estimator/cam_image" />
        <param name="autosize" value="true" />
    </node>-->

</launch>